.github/workflows/react_tests.yml

```diff
@@ -0,0 +1,69 @@
+name: React tests
+
+on:
+  push
+  pull_request:
+    branches: [master]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - repo: katello
+            org: laviro
+            ref: feat/foreman_js_link
```

Comment:
opened a PR to katello: https://github.com/Katello/katello/pull/8800 to support `link`